beta/nncf/tensorflow/sparsity/magnitude/algorithm.py

```diff
@@ -120,7 +120,12 @@ def __init__(self, target_model, config, op_names):
 
         sparsity_init = config.get('sparsity_init', 0)
         params['sparsity_init'] = sparsity_init
-        scheduler_cls = SPARSITY_SCHEDULERS.get(params.get('schedule', 'polynomial'))
+        scheduler_type = params.get('schedule', 'polynomial')
+
+        if scheduler_type == 'adaptive':
+            raise TypeError('Magnitude sparsity algorithm do not support adaptive scheduler')
```

Comment:
```suggestion
            raise ValueError('Magnitude sparsity algorithm do not support adaptive scheduler')
```

Reply:
I took this error from the PyTorch side. Do we need to fix it in the PyTorch too?

Reply:
Done